backend/api/appsettings.Development.json

```diff
@@ -19,5 +19,13 @@
     }
   },
   "AllowedHosts": "*",
-  "AllowedOrigins": "http://localhost:3001"
+  "AllowedOrigins": "http://localhost:3001",
+  "Mqtt": {
+    "Host": "localhost",
+    "Port": 1883,
+    "Username": "mosquitto",
+    "Topics": [
+      "#"
+    ]
+  }

```

Comment:
Have you seen these errors? I don't think subscribing to a wildcard is a very good idea. 
![image](https://user-images.githubusercontent.com/14331473/170004907-03197381-b03c-4404-864b-6b6689e7c222.png)

Should be mentioned that in my case this is a freshly started broker which seems to have some memory. Maybe there are some builtin topics too?

Reply:
Fixed by using more specific topics.